src/Sfa.Tl.ResultsAndCertification.Web/Controllers/AdminDashboardController.cs

```diff
@@ -560,7 +560,36 @@ public async Task<IActionResult> RemoveAssessmentEntryCoreAsync(AdminRemovePathw
             }
 
             await _cacheService.SetAsync(CacheKey, model);
-            return RedirectToRoute(RouteConstants.PageNotFound); // TODO: Redirect to review page
+            return RedirectToRoute(RouteConstants.AdminReviewRemoveCoreAssessmentEntry, new { registrationPathwayId = model.RegistrationPathwayId });
+        }
+
+        [HttpGet]
+        [Route("admin/review-remove-assessment-entry-core/{registrationPathwayId}", Name = RouteConstants.AdminReviewRemoveCoreAssessmentEntry)]
+        public async Task<IActionResult> AdminReviewRemoveCoreAssessmentEntryAsync(int registrationPathwayId)
```

Comment:
The parameter is not used at all in the method

Reply:
Updated